src/components/List.js

```diff
@@ -157,17 +157,17 @@ const List = ({ token }) => {
                 <i class="fas fa-cart-plus"></i>
                 Add Item
               </a>
-              <label htmlFor="search-bar">Filter Items</label>
-              <br />
-              <input
-                type="text"
-                name="search-bar"
-                id="search-bar"
-                placeholder="Start typing here..."
-                value={searchInput}
-                onChange={handleSearchChange}
-              />
-              <input type="reset" onClick={handleClear} />
+              <Box display="flex" alignItems="center">
+                <TextField
+                  name="search-bar"
+                  id="search-bar"
```

Comment:
Is this id being used? I don't see it being referenced in the List.css file.

Reply:
👍🏽  We can delete the id there